src/components/App.js

```diff
@@ -0,0 +1,15 @@
+// import React, { Component } from "react";
+
+// import '../styles/App.css';
+
+// class App extends Component {
+//     render() {
```

Comment:
Why is this file commented out? please work on this.

Reply:
I have worked on it by uncommenting the contents of the file